### ISSUE TITLE
[Launcher] Missing MSBuild installation

### ISSUE THREAD

**Issue by Eideren** [xenko3d/xenko#493]
**Release Type**: Official Release: launcher

**Version**: latest

**Platform(s)**: Launcher

**Describe the bug**
The launcher doesn't appear to correctly check/install msbuild, multiple users have been having this issue.

**To Reproduce**
1. Not having MSBuild installed
2. Run xenko installer+launcher.

**Expected behavior**
Install automatically MSBuild with at least the basic requirements to build a game on desktop, if that is not yet possible, present users with a guide to install the requirements themselves.

**Comment by mostafa901** [comment]
yep same here, with version 3.1.0.1 ... is there any workaround to start exploring ??

**Comment by Eideren** [comment]
@mostafa901 Yes, all you have to do is install MSBuild through here:
https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16
When the workloads window pops up select .NET desktop build tools, scroll down and select .NET Core build tools as well and you should be good to go.

**Comment by mostafa901** [comment]
Perfect, thanks a lot.

**Comment by xen2** [comment]
Logic to check MSBuild during installation is here:
https://github.com/xenko3d/xenko/blob/master/sources/tools/Xenko.PackageInstall/Program.cs

If it happened to someone, please let me know if Visual Studio was installed or not to more easily check which case to test on a VM.
Any details/repro would be useful!
